netutils/mqttc/Kconfig

```diff
@@ -25,12 +25,38 @@ config NETUTILS_MQTTC_EXAMPLE_STACKSIZE
 
 endif
 
+config NETUTILS_MQTTC_VERSION
+	string "MQTT-C Version"
+	default "1.1.5"
+
+endif	
+
 config NETUTILS_MQTTC_WITH_MBEDTLS
-	bool "Enable MQTT-C with mbedtls"
+	tristate "Enable MQTT-C with mbedtls"
 	default n
+	---help---
+		Enable MQTT-C with mbedtls
 
-config NETUTILS_MQTTC_VERSION
+if NETUTILS_MQTTC_WITH_MBEDTLS
+
+config NETUTILS_MQTTC_EXAMPLE_WITH_MBEDTLS
+	tristate "Enable MQTT-C example with mbedtls"
+	default n
+	---help---
+		Enable MQTT-C example with mbedtls
+
+if NETUTILS_MQTTC_EXAMPLE_WITH_MBEDTLS
+
+config NETUTILS_MQTTC_EXAMPLE_STACKSIZE_WITH_MBEDTLS
+	int "Task's stack size"
+	default 8192
+
+endif
+
+config NETUTILS_MQTTC_WITH_MBEDTLS_VERSION
```

Comment:
why need two version config?

Reply:
I assumed a user may want to have different MQTT-C versions for each of the two example apps. This is a corner case of course, but may be useful.

Reply:
but the build doesn't support to download multiple package.

Reply:
Was not aware of this, so will remove the additional version config